src/leaflet.draw.css

```diff
@@ -152,6 +152,10 @@
 .leaflet-draw-toolbar .leaflet-draw-draw-polyline {
 	background-position: -2px -2px;
 }
+.leaflet-draw-toolbar .leaflet-draw-draw-curve {
+	background-image: url('images/bezier.png') !important;
```

Comment:
`!important` should be avoided

Reply:
I will integrate the new icon into the spritesheet.